docs/design/fabric-v2+/fpc-registration.puml

```diff
@@ -40,7 +40,7 @@ end note
 
 Admin -> Admin: Attestation_Params <- <.e.g, SPID, sig-rl, ....>
 
-Admin -> Peer_CLI ++: lifecycle chaincode\n createenclave (CC_Id, Attestation_Params)
+Admin -> Peer_CLI ++: lifecycle chaincode\n createenclave (CC_Id, Attestation_Params, Peer_Endpoint)
```

Comment:
I don't have a better solution for this "peerendpoint", but it would certainly be better if the peer itself could supply it -- rather than an external entity.